nasa-gibs/worldview
# WV-2958 Additional Vegetation Indices

## Description
This adds new customizations to the customizable HLS layers. Added are expressions for EVI, SAVI, MSAVI, NBR, NBR2, and TVI. Also added are a new separate layer for each of the above 6 expressions, 1 each for Landsat and Sentinel.

## How To Test
1. `git checkout wv-2958-more-veg-indices`
2. `npm ci`
3. `npm run watch`
4. Open a fresh instance of Worldview
5. Add one of the customizable HLS layers, either `HLS_Customizable_Landsat` or `HLS_Customizable_Sentinel`
6. Zoom in far enough to see the layer, and pick a date where imagery shows up onscreen
7. Open the layer options, click 'Customize', and select one of the new expression options
8. Verify that the layer imagery looks correct for the option selected
9. Repeat for all other newly added expressions
10. Repeat also for the other customizable HLS layer that wasn't previously picked, either Landsat or Sentinel
11. Check through each of the 12 non-customizable layers for the 6 new expressions, and verify they look and work correctly as well
